ydb/library/yql/core/common_opt/yql_co_simple1.cpp

```diff
@@ -70,6 +70,42 @@ TExprNode::TPtr ExpandPgNot(const TExprNode::TPtr& input, TExprContext& ctx) {
         .Build();
 }
 
+template <bool Bool>
```

Comment:
please pass normal bool argument instead of a template parameter

Reply:
See above https://github.com/ydb-platform/ydb/pull/2978#discussion_r1531787239